namespaces/live-1.cloud-platform.service.justice.gov.uk/hmpps-book-secure-move-frontend-staging/resources/ecr.tf

```diff
@@ -0,0 +1,19 @@
+module "ecr-repo" {
+  source = "github.com/ministryofjustice/cloud-platform-terraform-ecr-credentials?ref=3.1"
```

Comment:
Please use the latest release:
https://github.com/ministryofjustice/cloud-platform-terraform-ecr-credentials/releases